package.json

```
@@ -1,6 +1,8 @@
 {
   "name": "azure-rest-api-specs",
   "devDependencies": {
+    "@autorest/openapi-to-typespec": "0.10.12",
```

```suggestion
    "@autorest/openapi-to-typespec": "0.10.13",
```

good to sort this correctly, but delete the dupplicate below

---

updated.